lib/screenplay/pa_messages.ex

```diff
@@ -19,7 +20,8 @@ defmodule Screenplay.PaMessages do
     @type t :: %{
             optional(:state) => :all | :active | :future | :past,
             optional(:now) => DateTime.t(),
-            optional(:signs) => [String.t(), ...]
+            optional(:signs) => [String.t(), ...],
+            optional(:routes) => [String.t(), ...]
```

Comment:
Just to clarify, it will be the client's responsibility to break down the labels `Busway` and `Silver` into their specific route IDs?

Reply:
Probably! We could add some aliases on the server if we find that's helpful but probably its own thing imo.